src/models/transfer/facade.js

```diff
@@ -273,7 +277,7 @@ const saveTransferPrepared = async (payload, stateReason = null, hasPassedValida
       value: payload.ilpPacket
     }
 
-    const state = ((hasPassedValidation) ? Enum.TransferState.RECEIVED_PREPARE : Enum.TransferState.INVALID)
+    const state = ((hasPassedValidation) ? Enum.TransferState.RECEIVED_PREPARE : Enum.TransferState.ABORTED)
```

Comment:
Regarding this one - I think the intention was to use INVALID internally to easily identify requests for which validation failed. Here is the sequence diagram - https://github.com/mojaloop/docs/blob/develop/CentralServices/seq_diagrams/seq-prepare-1.1.1.a.svg

Reply:
@elnyry Invalid does not exist in the transferState table. This will fail because of a foreign key exception

Reply:
Its in the seeds here - https://github.com/mojaloop/central-ledger/blob/develop/seeds/transferState.js ?